packages/shared/src/dates.ts

```diff
@@ -28,6 +28,23 @@ export function daysBetween(start: string | Date, end: string | Date): number {
   return differenceInDays(getValidDate(end), getValidDate(start));
 }
 
+export function dateStringArrayBetweenDates(
+  start: Date,
+  end: Date,
+  truncate: boolean = true,
+  dayInterval: number = 1
+): string[] {
+  const dateArray: string[] = [];
+  let startTruncate = new Date(start);
+  if (truncate) {
+    startTruncate = new Date(start.toISOString().substring(0, 10));
+  }
+  for (let d = startTruncate; d <= end; d.setDate(d.getDate() + dayInterval)) {
+    dateArray.push(`'${d.toISOString().substring(0, 10)}'`);
+  }
+  return dateArray;
+}
```

Comment:
I'd probably add some test cases for this function.

Reply:
Will do.